lib/views/Account/V4Deposits/index.tsx

```diff
@@ -149,6 +157,20 @@ const DepositItem = (props: DepositItemsProps) => {
             theme: SquareButtonTheme.tealOutline
           }
         ]}
+        moreInfoViews={[
+          {
+            id: 'delegate',
+            view: () => (
+              <DelegateView
+                prizePool={prizePool}
+                balances={balances}
+                refetchBalances={refetchBalances}
+              />
+            ),
+            label: t('delegateDeposits', 'Delegate deposits'),
```

Comment:
Same here, singular vs plural

Reply:
Also would be great to have an icon on this button, as the button below it "Add to MetaMask" has an icon.